ProcessLib/LiquidFlow/LiquidFlowLocalAssembler.h

```diff
@@ -61,21 +62,23 @@ class LiquidFlowLocalAssembler : public LiquidFlowLocalAssemblerInterface
     using GlobalDimVectorType = typename ShapeMatricesType::GlobalDimVectorType;
 
 public:
-    LiquidFlowLocalAssembler(MeshLib::Element const& element,
-                             std::size_t const /*local_matrix_size*/,
-                             bool const is_axially_symmetric,
-                             unsigned const integration_order,
-                             int const gravitational_axis_id,
-                             double const gravitational_acceleration,
-                             LiquidFlowMaterialProperties& material_propertries)
+    LiquidFlowLocalAssembler(
+        MeshLib::Element const& element,
+        std::size_t const /*local_matrix_size*/,
+        bool const is_axially_symmetric,
+        unsigned const integration_order,
+        int const gravitational_axis_id,
+        double const gravitational_acceleration,
+        std::unique_ptr<LiquidFlowMaterialProperties> const&
+            material_propertries)
         : _element(element),
           _integration_method(integration_order),
           _shape_matrices(initShapeMatrices<ShapeFunction, ShapeMatricesType,
                                             IntegrationMethod, GlobalDim>(
               element, is_axially_symmetric, _integration_method)),
           _gravitational_axis_id(gravitational_axis_id),
           _gravitational_acceleration(gravitational_acceleration),
-          _material_properties(material_propertries)
+          _material_properties(*(material_propertries.get()))
```

Comment:
Changed back to ``_material_properties(material_propertries)`` because the argument type ``std::unique_ptr<LiquidFlowMaterialProperties> const&`` has been changed to ``LiquidFlowMaterialProperties const&`` according to you last comment.

PS: I made these two changes by amending the last commit.